### ISSUE TITLE
Result on ScanNet200 3D semantic label benchmark

### ISSUE THREAD

**Issue by dhgras** [Pointcept/PointTransformerV3#25]
How to train to obtain the following result on ScanNet200?
![image](https://github.com/Pointcept/PointTransformerV3/assets/42953352/c16c826c-12c8-4d6a-b8e4-67534b219776)

**Comment by Gofinge** [comment]
Hi, here are some suggestions:
1. Add validation set to the training set for submission.
2. Train three models separately and ensemble the predicted logic to one prediction (A common trick for perception challenges, but remember, don't use it to report model validation performance.)
I will write a document about step-by-step benchmark instruction later (When I have some time).

**Comment by dhgras** [comment]
> Hi, here are some suggestions:
> 
> 1. Add validation set to the training set for submission.
> 2. Train three models separately and ensemble the predicted logic to one prediction (A common trick for perception challenges, I think it was initially introduced to ScanNet benchmark by Stratified Transformer. But remember, don't use it to report model validation performance.)
> 3. For ScanNet, it may be a bit of cheating, using over-segmentation, which was also used in ScanNet annotation. (I think this trick was initially introduced by Mix3D, and then Mask3D and Swin3D etc, also use a similar trick. We have no choice, but also, please don't use it to report validation performance. I hope it can keep pure.
> 
> I will write a document about step-by-step benchmark instruction later (When I have some time).

Thanks for your reply! Regarding the second suggestion, do you mean to train three models with different training parameters? Then how to integrate the prediction logic into one prediction?

**Comment by Gofinge** [comment]
Kindly remind anyone who reads these instructions: **we do not use any trick for validation results. Also, please don't involve these tricks in validating benchmarks.**

**Comment by hfutdzx** [comment]
> Kindly remind anyone who reads these instructions: **we do not use any trick for validation results. Also, please don't involve these tricks in validating benchmarks.**

solid work！can you provide over-segmentation scheme codes？

**Comment by Gofinge** [comment]
Hi @hfutdzx , check here (https://github.com/ScanNet/ScanNet/tree/master/Segmentator)
